.github/CHANGELOG.md

```diff
@@ -1,16 +1,18 @@
-# Release 0.23.0-dev
-
-### New features since last release
-
-### Breaking changes
+# Release 0.23.0
 
 ### Improvements
+* Update builder and cuQuantum SDK support [(#10)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/10).
 
-### Documentation
+### Contributors
 
-### Bug fixes
+This release contains contributions from (in alphabetical order):
+Ali Asadi, and Lee James O'Riordan
```

Comment:
I think my name shouldn't be tagged in this one,

```suggestion
Lee James O'Riordan
```

Reply:
Did you not add a commit or two to fix something in this release?

Reply:
@maliasadi even contributing via code review suggestions is a contribution that should be listed!

Reply:
Yes, I did, but they were minor. :slightly_smiling_face: